packages/main/src/BusyIndicator.ts

```diff
@@ -143,7 +65,87 @@ const metadata = {
  * @public
  * @since 0.12.0
  */
+@customElement("ui5-busy-indicator")
+@languageAware
 class BusyIndicator extends UI5Element {
+	/**
+	 * Defines text to be displayed below the component. It can be used to inform the user of the current operation.
+	 * @type {string}
+	 * @name sap.ui.webc.main.BusyIndicator.prototype.text
+	 * @public
+	 * @defaultvalue ""
+	 * @since 1.0.0-rc.7
+	 */
+	@property()
+	text!: string;
+
+	/**
+	 * Defines the size of the component.
+	 *
+	 * <br><br>
+	 * <b>Note:</b>
+	 *
+	 * <ul>
+	 * <li><code>Small</code></li>
+	 * <li><code>Medium</code></li>
+	 * <li><code>Large</code></li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.main.types.BusyIndicatorSize}
+	 * @name sap.ui.webc.main.BusyIndicator.prototype.size
+	 * @defaultvalue "Medium"
+	 * @public
+	 */
+	@property({ type: BusyIndicatorSize, defaultValue: BusyIndicatorSize.Medium })
+	size!: BusyIndicatorSize;
+
+	/**
+	 * Defines if the busy indicator is visible on the screen. By default it is not.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.BusyIndicator.prototype.active
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	active!: boolean;
+
+	/**
+	 * Defines the delay in milliseconds, after which the busy indicator will be visible on the screen.
+	 *
+	 * @type {sap.ui.webc.base.types.Integer}
+	 * @name sap.ui.webc.main.BusyIndicator.prototype.delay
+	 * @defaultValue 1000
+	 * @public
+	 */
+	@property({ validator: Integer, defaultValue: 1000 })
+	delay!: number;
+
+	/**
+	 * Defines if the component is currently in busy state.
+	 * @private
+	 */
+	@property({ type: Boolean })
+	_isBusy!: boolean;
+
+	/**
+	 * Determines the content over which the component will appear.
+	 *
+	 * @type {Node[]}
+	 * @name sap.ui.webc.main.BusyIndicator.prototype.default
+	 * @slot
+	 * @public
+	 */
+	@slot({ "default": true, type: Node })
```

Comment:
When managedSlots is false for default slot without property name we use only comments.